content/en/docs/troubleshooting/_index.md

```diff
@@ -141,6 +141,12 @@ This issue occurs when logging in using a service account along with your platfo
 
 Please create a ticket in the [Axway Support Portal](https://support.axway.com/) to have your organization provisioned with the appropriate subscription.
 
+## TypeError: Expected code for interactive authentication to be a non-empty string
+
+This is an internal error that can occur when the Axway CLI refreshes an expired access token while running an extension CLI command that uses an incompatible dependency.
+
+To resolve the issue, update to the latest Axway CLI and update all your CLI extensions by running `axway pm update`.
```

Comment:
How about this.... To resolve this issue, run `axway pm update` to update your Axway CLI and CLI extensions to the latest version.